src/main/java/com/nutomic/syncthingandroid/syncthing/SyncthingService.java

```diff
@@ -429,9 +432,19 @@ public IBinder onBind(Intent intent) {
      */
     @Override
     public void onDestroy() {
-        super.onDestroy();
-        Log.i(TAG, "Shutting down service");
-        shutdown();
+//        super.onDestroy();    Call of super not required -> https://developer.android.com/guide/components/services.html
```

Comment:
I'd prefer to use a Javadoc that describes what this method does, instead of this comment (which isn't really helpful).